SportsHub/SportsHub.Api/Controllers/RegisterController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using SportsHub.AppService.Authentication;
+using SportsHub.AppService.Authentication.Models.DTOs;
+
+namespace SportsHub.Api.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class RegisterController : ControllerBase
+    {
+        private IAuthenticationService _authenticationService;
```

Comment:
let's think about formatting as well. Add an empty line between the private variable and the constructor. Look through the changes and fix formatting: return should be after an empty line (except {}), separate logic blocks, comments should be after an empty line or start of the method, etc.